apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,192 @@ const data: FoundAnimalFlow = {
         ],
       },
     },
+    {
+      name: "Cat",
+      flow: {
+        prompt: [
+          "Does the cat appear to be sick, injured, in danger, or a nursing kitten with no mama in sight?",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: macros.general.rehab("the cat"),
+            },
+          },
+          {
+            name: "No",
+            flow: {
+              prompt: [
+                "Has the cat been outside for over 24 hours?",
+              ],
+              options: [
+                {
+                  name: "Yes",
+                  flow: {
+                    prompt: [
+                      "Many cats that people spot in neighborhoods aren't in need of immediate help. Instead, they are pets who spend some or all of their time outdoors.",
+                      "Check for a collar, if the cat has one try and get in contact with the owner, If the cat has no collar you can take the cat to the nearest animal shelter to check for a microchip",
+                      "If the cat does not have a microchip, assess whether the cat seems healthy (not injured or visibly ill) and safe (in a secure location, away from any danger) We want a healthy and safe cat to stay where it is while you attempt to find the owner",
```

Comment:
I feel like this part doesn't really make sense. We say "while you attempt to find the owner", but never actually tell the user to find the owner?

We say "assess whether the cat seems healthy (not injured or visibly ill) and safe (in a secure location, away from any danger)", but don't give any guidance if the cat is unhealthy? We've already asked further up in the flow if the cat seems unwell, so maybe this can be removed completely?

I think this last bit might just want to be something along the line of "If the cat does not have a microchip, leave the cat where it is and attempt to locate the owner (perhaps ask neighbours, or leave up flyers)", and then perhaps guidance for contacting a rehabber if the cat appears to be feral/unowned?